client/state/themes/test/actions.js

```diff
@@ -880,8 +878,8 @@ describe( 'actions', () => {
 				expect( spy ).to.have.been.calledWith( sinon.match.func );
 			} );
 		} );
-
 		test( 'should dispatch failure on error', () => {
+			/* eslint-disable jest/no-conditional-expect */
```

Comment:
**Note:**
These test changes are unrelated to this PR. They're meant to fix failing eslint rules that were making teamCity CI angry.